app/models/payments/ChargeType.scala

```diff
@@ -476,21 +508,25 @@ object ChargeType extends LoggerUtil {
     VatManualLPP -> VatManualLPPLPI,
     AAQuarterlyInstalments -> VatAAQuarterlyInstalLPI,
     AAMonthlyInstalment -> VatAAMonthlyInstalLPI,
-    VatReturn2ndLPP -> VatReturn2ndLPPLPI
+    VatReturn2ndLPP -> VatReturn2ndLPPLPI,
+    VatErrorCorrection1stLPP -> VatErrorCorrection1stLPPLPI,
+    VatErrorCorrection2ndLPP -> VatErrorCorrection2ndLPPLPI
   )
 
   val penaltyChargeMappingLPP1: Map[ChargeType, ChargeType] = Map(
     ReturnDebitCharge -> VatReturn1stLPP,
     VatProtectiveAssessmentCharge -> VatPA1stLPP,
     AACharge -> VatAA1stLPP,
-    AAReturnDebitCharge -> VatAAReturnCharge1stLPP
+    AAReturnDebitCharge -> VatAAReturnCharge1stLPP,
+    VatECDefaultInterestCharge -> VatErrorCorrection1stLPP
```

Comment:
The principal charge type that this and the LPP2 below relate to is `VAT EC Debit Charge`